hail/python/hail/experimental/db.py

```diff
@@ -312,7 +312,7 @@ class DB:
     >>> db = hl.experimental.DB(region='us', cloud='gcp')
     """
 
-    _valid_key_properties = {'gene', 'unique'}
+    _valid_key_properties = {'gene', 'unique', 'non_unique'}
```

Comment:
I think we shouldn't do this. If `'unique'` is not in the properties, that implies non_unique. If `'unique'` is missing from the key properties, we annotate a list of matching results.

Reply:
Put another way: what does it mean if neither unique or non_unique are in the valid key properties.

Reply:
Gotcha, yeah I just wasn't sure how explicit to be in the json with non-unique keys. Should be good now.